models/models/models.py

```diff
@@ -0,0 +1,159 @@
+from datetime import datetime
+from os import environ
+from typing import List
+
+import boto3
+from boto3.dynamodb.conditions import Attr, Key
+
+DB = boto3.resource('dynamodb')
+EVENT_TABLE = environ['EVENT_TABLE']
+PRODUCT_TABLE = environ['PRODUCT_TABLE']
+
+
+class Event:
+    def __init__(self, event_id, processing_start=None, processing_end=None, wkt=None, **kwargs):
+        self.event_id = event_id
+        self.processing_start = processing_start
+        self.processing_end = processing_end
+        self.wkt = wkt
+        self.extras = kwargs
+
+    def __eq__(self, other):
```

Comment:
these comparissons could just be 
```
self.to_dict() == other.to_dict()
```
the benefit to this mess is that it allows comparison of Events to dictionary representation of events